pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dto/UserDto.java

```diff
@@ -37,6 +41,14 @@ public class UserDto {
     @ColumnName("expires_at")
     Long expiresAtMillis;
 
+    public Optional<String> getAuth0UserId() {
+        return Optional.ofNullable(auth0UserId);
+    }
+
+    public Optional<String> getEmail() {
+        return Optional.ofNullable(email);
+    }
```

Comment:
Do you think it is useful? I see that this method is being used only in one place - UserCreationRoute class.

The construction
```
model.getEmail().orElse(null);
```
looks a bit less readable for me that 
```
Optional.ofNullable(model.getEmail()).orElse(null);
```
or simple ternary operator
```
model.getEmail() == null ? null : model.getEmail();
```

Anyway, it's just a matter of taste, so if you believe returning optional is better, then let's leave it as is.